### DISCUSSION TITLE
Trying to set up load testing using k6

### DISCUSSION THREAD

**Discussion by annismckenzie** [zitadel/zitadel#8934] [Q&A] [answered]
We're currently trying to set up load tests using k6. The repository already contains a package `load-test` at https://github.com/zitadel/zitadel/tree/main/load-test for exactly that.

Unfortunately, the code uses the repository https://github.com/zitadel/xk6-modules which seems to be private as it results in a 404. We want to avoid writing our own if at all possible.

**Comment by fforootd** [comment]
@adlerhurst I think you wanted to publish that repository, right?

**Reply by adlerhurst** [reply]
yes need to prepare the repo for making it public and then it will get available.

**Reply by arminbuerkle** [reply]
Any progress / ETA on this? We're still interested in using this to run our own load tests.

**Reply by adlerhurst** [reply, marked as answer]
We have just published https://github.com/zitadel/xk6-modules which is required by the load tests. So you should now be able to run the load tests
